src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-revisions-controller.php

```diff
@@ -419,133 +342,36 @@ public function get_item_schema() {
+		$schema['properties'] = array_merge( $schema['properties'], $parent_schema['properties'] );
 
-				// Adds settings and styles from the WP_REST_Global_Styles_Controller parent schema.
-				'styles'       => array(
-					'description' => __( 'Global styles.' ),
-					'type'        => array( 'object' ),
-					'context'     => array( 'view', 'edit' ),
-				),
-				'settings'     => array(
-					'description' => __( 'Global settings.' ),
-					'type'        => array( 'object' ),
-					'context'     => array( 'view', 'edit' ),
-				),
-			),
-		);
+		unset( $schema['properties']['guid'] );
+		unset( $schema['properties']['slug'] );
+		unset( $schema['properties']['meta'] );
+		unset( $schema['properties']['content'] );
+		unset( $schema['properties']['title'] );
 
 		$this->schema = $schema;
 
 		return $this->add_additional_fields_schema( $this->schema );
 	}
 
 	/**
-	 * Checks if a given request has access to read a single global style.
-	 *
-	 * @since 6.3.0
-	 *
-	 * @param WP_REST_Request $request Full details about the request.
-	 * @return true|WP_Error True if the request has read access, WP_Error object otherwise.
-	 */
-	public function get_item_permissions_check( $request ) {
-		$post = $this->get_parent( $request['parent'] );
-		if ( is_wp_error( $post ) ) {
-			return $post;
-		}
-
-		/*
-		 * The same check as WP_REST_Global_Styles_Controller::get_item_permissions_check.
-		 */
-		if ( ! current_user_can( 'read_post', $post->ID ) ) {
-			return new WP_Error(
-				'rest_cannot_view',
-				__( 'Sorry, you are not allowed to view revisions for this global style.' ),
-				array( 'status' => rest_authorization_required_code() )
-			);
-		}
-
-		return true;
-	}
-
-	/**
-	 * Gets the parent post, if the ID is valid.
-	 *
-	 * Duplicate of WP_REST_Revisions_Controller::get_parent.
+	 * Retrieves the query params for collections.
+	 * Removes params that are not supported by global styles revisions.
 	 *
-	 * @since 6.3.0
+	 * @since 6.5.0
 	 *
-	 * @param int $parent_post_id Supplied ID.
-	 * @return WP_Post|WP_Error Post object if ID is valid, WP_Error otherwise.
+	 * @return array Collection parameters.
 	 */
-	protected function get_parent( $parent_post_id ) {
-		$error = new WP_Error(
-			'rest_post_invalid_parent',
-			__( 'Invalid post parent ID.' ),
-			array( 'status' => 404 )
-		);
-
-		if ( (int) $parent_post_id <= 0 ) {
-			return $error;
-		}
-
-		$parent_post = get_post( (int) $parent_post_id );
-
-		if ( empty( $parent_post ) || empty( $parent_post->ID )
-			|| $this->parent_post_type !== $parent_post->post_type
-		) {
-			return $error;
-		}
-
-		return $parent_post;
+	public function get_collection_params() {
+		$query_params = parent::get_collection_params();
+		unset( $query_params['exclude'] );
+		unset( $query_params['include'] );
+		unset( $query_params['search'] );
+		unset( $query_params['order'] );
+		unset( $query_params['orderby'] );
+		return $query_params;
```

Comment:
Nit: just use one `unset()` call for all of these:

`unset( $query_params['exclude'], $query_params['include'], ... )`

Reply:
Just in case @spacedmonkey is too busy, I've created a branch over in https://github.com/WordPress/wordpress-develop/pull/6573 with these updates as I can't edit this branch.

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/6105/commits/c62fed9b8f8c1403f7ee4c06151d833a9b8fcd3f